clutter/SplitScreenScanner
# Scan History TableView

## Pivotal Tickets
 - https://www.pivotaltracker.com/story/show/156144492
 - https://www.pivotaltracker.com/story/show/156144389

## Feature Images
Now when you start scanning in the test app you will be presented with the split barcode scanner and a TableView asking you to scan your first item.
![img_0055 2](https://user-images.githubusercontent.com/10718632/38571569-b60f0bc2-3ca5-11e8-9c41-c37283f69be6.PNG)

After scanning an item you will see the history of that scan appear in the table view just below the scanner. Each scan has a random chance of being a success, a warning, or an error.
![img_0054 2](https://user-images.githubusercontent.com/10718632/38571805-64dccde2-3ca6-11e8-86ad-673eb7f5bf06.PNG)


## What To Test?
- Open up the test app and click the "Start Scanning" button.
- Make sure the "Scan an item to start loading" cell exists and disappears on first scan.
- Test scanning a few barcodes making sure the history of each barcode is correct.
